src/Package/Impl/Help/VignetteCodeColorBuilder.cs

```diff
@@ -0,0 +1,96 @@
+            new CssCodeProperty("ss", PredefinedClassificationTypeNames.String),
+            new CssCodeProperty("co", PredefinedClassificationTypeNames.Comment),
+            new CssCodeProperty("fu", PredefinedClassificationTypeNames.Keyword),
+            new CssCodeProperty("va", PredefinedClassificationTypeNames.Identifier),
+            new CssCodeProperty("ot", PredefinedClassificationTypeNames.Other),
+            new CssCodeProperty("cn", PredefinedClassificationTypeNames.Number),
+            new CssCodeProperty("dv", PredefinedClassificationTypeNames.Number),
+            new CssCodeProperty("cf", PredefinedClassificationTypeNames.Keyword),
+            new CssCodeProperty("op", PredefinedClassificationTypeNames.Operator),
+            new CssCodeProperty("pp", PredefinedClassificationTypeNames.PreprocessorKeyword),
+        };
+
+        private static readonly CssCodeProperty[] _prePropertyMap = new CssCodeProperty[] {
+            new CssCodeProperty("keyword", PredefinedClassificationTypeNames.Keyword),
+            new CssCodeProperty("number", PredefinedClassificationTypeNames.Number),
+            new CssCodeProperty("string", PredefinedClassificationTypeNames.String),
+            new CssCodeProperty("literal", PredefinedClassificationTypeNames.Literal),
+            new CssCodeProperty("comment", PredefinedClassificationTypeNames.Comment),
+            new CssCodeProperty("identifier", PredefinedClassificationTypeNames.Identifier),
+            new CssCodeProperty("operator", PredefinedClassificationTypeNames.Operator),
+            new CssCodeProperty("paren", PredefinedClassificationTypeNames.Operator),
+        };
+
+        private readonly IRInteractiveWorkflowProvider _workflowProvider;
+        private readonly IClassificationFormatMapService _formatMapService;
+        private readonly IClassificationTypeRegistryService _classificationRegistryService;
+
+        [ImportingConstructor]
+        public VignetteCodeColorBuilder(
+            IRInteractiveWorkflowProvider workflowProvider,
+            IClassificationFormatMapService cfms, 
+            IClassificationTypeRegistryService ctrs) {
+            _workflowProvider = workflowProvider;
+            _formatMapService = cfms;
+            _classificationRegistryService = ctrs;
+        }
+
+        public string GetCodeColorsCss() {
+            var sb = new StringBuilder();
+            var workflow = _workflowProvider.GetOrCreate();
+            if (workflow.ActiveWindow?.TextView != null) {
+                var map = _formatMapService.GetClassificationFormatMap(workflow.ActiveWindow.TextView);
+                foreach (var cssCodeProp in _cssPropertyMap) {
+                    var props = map.GetTextProperties(_classificationRegistryService.GetClassificationType(cssCodeProp.ClassificationTypeName));
+
+                    sb.AppendLine(Invariant($"code > span.{cssCodeProp.CssClassName} {{"));
```

Comment:
So these 4 lines can be combined into one:
```{csharp}
$@"code > span.{cssCodeProp.CssClassName} {{
    color: {CssColorFromBrush(props.ForegroundBrush)};
}}
";
```
I think it will be easier to read this way